src/styles/animation.scss

```diff
@@ -0,0 +1,10 @@
+@import './variables.scss';
+
+.calendar__day {
+  transition: transform $transition-duration, background-color $transition-duration;
```

Comment:
Ensure that `$transition-duration` is defined in your `variables.scss` file. Undefined variables will cause compilation errors.